tasks/label_analysis.py

```diff
@@ -410,17 +419,39 @@ def calculate_final_result(
                 commit=commit_sha,
             ),
         )
+        self.metrics_context.log_simple_metric(
+            "label_analysis.tests_saved_count", len(all_report_labels)
+        )
+        self.metrics_context.log_simple_metric(
+            "label_analysis.requests_with_requested_labels",
+            float(requested_labels is not None),
+        )
```

Comment:
i think you should still be able to use `attempt_log_simple_metric()` here despite this not being in an async function? as long as you're running inside an asyncio loop it should let you. i guess if you're invoking this from a synchronous test it will get mad that there isn't a runtime